pkgs/development/ocaml-modules/smtml/default.nix

```diff
@@ -25,7 +25,7 @@
 
 buildDunePackage rec {
   pname = "smtml";
-  version = "0.7.0";
+  version = "0.8.0";
```

Comment:
When you change this, you must also adapt the `hash` in the `src` attribute below. If you cannot guess the value (by divine inspiration or whatever), start with an empty string, and use `nix` to build your package: it will kindly give you the correct value.